Makefile

```diff
@@ -1,21 +1,30 @@
+.PHONY: all
 all:
-	@jbuilder build @install @DEFAULT
+	@dune build @install @examples
 
+.PHONY: run-examples
+run-examples:
+	dune exec examples/filtering.exe < examples/filtering.json
+
+.PHONY: test
 test:
-	@jbuilder runtest
+	@dune runtest --force
```

Comment:
The `test` target can be misleading as it currently neither run nor build anything.

I'd really like to write a test suite for `yojson` but in the meantime I think it would be better to remove this. What do you think?